src/blockdata/transaction.rs

```diff
@@ -220,12 +220,83 @@ impl Default for TxIn {
         TxIn {
             previous_output: OutPoint::default(),
             script_sig: Script::new(),
-            sequence: u32::max_value(),
+            sequence: Sequence::MAX,
             witness: Witness::default(),
         }
     }
 }
 
+/// Bitcoin transaction input sequencing number.
+/// 
+#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Debug, Hash)]
```

Comment:
`Ord` is fine here because it's not disable bit but disable value. Some old consensus/mempool rules even compared sequences and that's OK.

However if we allow getting `enum RelativeLockTime` from `Sequence` that enum will not be `Ord`